src/encoded/upgrade/dataset.py

```diff
@@ -364,3 +364,21 @@ def dataset_16_17(value, system):
 def dataset_17_18(value, system):
     if value['status'] == "started":
         value['status'] = "in progress"
+
+
+@upgrade_step('experiment', '18', '19')
```

Comment:
I suppose you can't inherit dependency from parent dataset. Copying this into all of the children objects and keeping them updated seems like a pain.